code/obj/machinery/computer/QM_supply.dm

```diff
@@ -1063,10 +1069,22 @@ var/global/datum/cdc_contact_controller/QM_CDC = new()
 		if(RC.flavor_desc) src.temp += "[RC.flavor_desc]<br><br>"
 		src.temp += "[RC.requis_desc]"
 		if(RC.req_class == AID_CONTRACT)
-			src.temp += "URGENT - Cannot Be Reserved"
+			src.temp += "URGENT - Cannot Be Reserved<br>"
 		else
-			src.temp += "<A href='[topicLink("pin_contract","\ref[RC]")]'>[RC.pinned ? "Unpin Contract" : "Pin Contract"]</A>"
-
+			src.temp += "<A href='[topicLink("pin_contract","\ref[RC]")]'>[RC.pinned ? "Unpin Contract" : "Pin Contract"]</A><br>"
+		src.temp += "<A href='[topicLink("print_req","\ref[RC]")]'>Print List</A>"
+
+/obj/machinery/computer/supplycomp/proc/print_requisition(var/datum/req_contract/contract)
+	src.printing = 1
+	playsound(src.loc, "sound/machines/printer_thermal.ogg", 60, 0)
+	SPAWN_DBG(2 SECONDS)
+		var/obj/item/paper/thermal/P = new/obj/item/paper/thermal(src.loc)
+		P.info = "<font face='System' size='2'><center>REQUISITION CONTRACT MANIFEST<br>"
```

Comment:
You seem to be missing an end font tag unless I'm blind

Reply:
Huh, sure enough. Changed.